libs/draft-comment/src/draft-comment.service.ts

```diff
@@ -137,6 +166,28 @@ export class DraftCommentService {
     return true;
   }
 
+  async delete(id: string, accountId: string): Promise<boolean> {
+    const draftComment = await this.draftCommentRepository.findOne(id);
+
+    if (!draftComment) {
+      throw new NotFoundException(`Draft comment ${id} does not exist`);
+    }
+
+    const accountPermissions =
+      await this.proposalService.getAccountPermissionByDao(
+        draftComment.daoId,
+        accountId,
+      );
+
+    if (draftComment.author !== accountId && !accountPermissions.isCouncil) {
+      throw new ForbiddenException('Account is not the author or council');
+    }
+
+    await this.draftCommentRepository.delete(draftComment);
+
+    return true;
```

Comment:
It would be better to return DeleteResult instead of a bool here.

Reply:
Done